go/fn/object_test.go

```diff
@@ -414,7 +414,7 @@ func TestSetNestedFields(t *testing.T) {
 func TestInternalAnnotationsUntouchable(t *testing.T) {
 	o := NewEmptyKubeObject()
 	// Verify the "upstream-identifier" annotation cannot be changed via SetNestedStringMap
-	o.SetNestedStringMap(map[string]string{"owner": "kpt"}, "metadata", "annotations")
+	o.SetNestedStringMapOrDie(map[string]string{"owner": "kpt"}, "metadata", "annotations")
```

Comment:
If `SetNestedStringMapOrDie` panic here, does it crash the test or we recover somewhere?

Reply:
I expect it to crash. 
Basically I want the unittests and examples to more showing the `OrDie` usage rather than letting users deal with Found and Error.